content/en/aws/s3/index.md

```diff
@@ -58,3 +58,81 @@ As a special case in LocalStack, leaving out `.s3.<region>` also works for the `
 
 All other requests will be considered path-style requests.
 {{% /alert %}}
+
+## Storage Configuration
+
+Localstack Pro can be configured to store S3 files in specific locations on the filesystem. This can be useful in many different ways, including moving multiple files into place in one or more S3 buckets without needing to use `awslocal` to upload them. Localstack will use the path(s) specified in the `S3_DIRS` environment variable to store files within the container. Paired with a docker mount, this can be used to store files in the path of choice on the filesystem.
+
+Paths may be specified in two different configurations. Specifying only a path will store all S3 buckets in that path:
+
+```
```

Comment:
Specify the syntax highlighting: `sh` / `bash` in this case

Reply:
Fixed in latest commit. 👍